scripts/changelog-script.sh

```diff
@@ -12,26 +22,29 @@ fi
 
 MIRROR="https://mirror.openshift.com/pub/openshift-v4/clients/odo/$2/"
 INSTALLATION_GUIDE="https://docs.openshift.com/container-platform/latest/cli_reference/developer_cli_odo/installing-odo.html"
-GIT_TREE="https://github.com/openshift/odo/tree/$2"
-FULL_CHANGELOG="https://github.com/openshift/odo/compare/$1...$2"
-
-echo -e "
-# Release of $2
-
-## [$2]($GIT_TREE) ($(date '+%Y-%m-%d'))
-
-[Full Changelog]($FULL_CHANGELOG)
-
--- COPY CONTENT FROM Changelog.md here --
-
-# Installation of $2
 
+echo -e "# Installation of $2
 To install odo, follow our installation guide at [docs.openshift.com]($INSTALLATION_GUIDE)
+After each release, binaries are synced to [mirror.openshift.com]($MIRROR)" > /tmp/base
+
+github_changelog_generator \
+--max-issues 500 \
+--user openshift \
+--project odo \
+--no-issues \
+-t $GITHUB_TOKEN \
+--since-tag $1 \
+--future-release $2 \
+--base /tmp/base \
+--output /tmp/changelog \
+--exclude-labels "lifecycle/rotten,duplicate,question,invalid,wontfix" \
```

Comment:
I don't think we have these labels in odo repo: `duplicate`, `question`, `invalid`, `wontfix`. Do we?

Reply:
There are the default labels, so i chose to keep them